RecoBTag/SoftLepton/src/ElectronTagger.cc

```diff
@@ -11,13 +11,19 @@ float ElectronTagger::discriminator(const TagInfoHelper & tagInfo) const {
   // default value, used if there are no leptons associated to this jet
   float bestTag = - std::numeric_limits<float>::infinity();
   const reco::CandSoftLeptonTagInfo & info = tagInfo.get<reco::CandSoftLeptonTagInfo>();
+
+  std::default_random_engine random;
```

Comment:
what's the meaning of default_random_engine?
Is it reproducible across platforms?
Looking at www.open-std.org/jtc1/sc22/wg21/docs/papers/2013/n3551.pdf (just google search) 
```
default_random_engine, an alias for an engine type selected by the
library vendor
... distinct vendors are free to select differently
... code that uses this alias need not
generate identical sequences across implementations
```

TRandom3 is Mersenne twister engine; I'm not sure this randomness quality is really required here, but at least the specifications for it are defined.